prdoc/pr_6643.prdoc

```diff
@@ -0,0 +1,44 @@
+title: Added fallback_max_weight to Transact for sending messages to V4 chains
+doc:
+- audience: Runtime Dev
+  description: |-
+    Removing the `require_weight_at_most` parameter in V5 Transact had only one problem. Converting a message from V5 to V4 to send to chains that didn't upgrade yet. The conversion would not know what weight to give to the Transact, since V4 and below require it.
```

Comment:
```suggestion
    Removing the `require_weight_at_most` parameter in V5 Transact introduced a problem when converting a message from V5 to V4 to send to chains that didn't upgrade yet. The local chain doesn't know how to decode call for remote chain so it can't automatically populate `require_weight_at_most` required by V4 Transact.
```